ui/src/dashboards/components/VEOContents.tsx

```diff
@@ -40,8 +38,9 @@ interface OwnProps {
 type Props = OwnProps & StateProps & DispatchProps
 
 class VEOContents extends PureComponent<Props, {}> {
-  public componentDidMount() {
-    this.props.executeQueries()
+  public async componentDidMount() {
+    const {executeQueries} = this.props
+    await executeQueries()
```

Comment:
Is there any need to make `componentDidMount` return a promise rather than void? afaik the result of `componentDidMount` is always discarded

Reply:
not needed!  ill pull it out